modules/spring-config/src/test/java/org/mule/runtime/config/internal/factories/FlowRefFactoryBeanTestCase.java

```diff
@@ -273,63 +289,86 @@ private Processor getFlowRefProcessor(FlowRefFactoryBean factoryBean) throws Exc
   @Test
   public void dynamicFlowRefDoesNotExist() throws Exception {
     doReturn(new TypedValue<>("other", STRING)).when(expressionManager).evaluate(eq(DYNAMIC_NON_EXISTANT), eq(DataType.STRING),
-            eq(NULL_BINDING_CONTEXT), any(CoreEvent.class),
-            any(ComponentLocation.class), eq(true));
+                                                                                 eq(NULL_BINDING_CONTEXT), any(CoreEvent.class),
+                                                                                 any(ComponentLocation.class), eq(true));
 
     expectedException.expect(instanceOf(RoutePathNotFoundException.class));
     getFlowRefProcessor(createFlowRefFactoryBean(DYNAMIC_NON_EXISTANT, "flow", applicationContext)).process(testEvent());
   }
 
-  @Test
+  @Test()
+  @Feature(AllureConstants.SpringConfiguration.SPRING_CONFIGURATION)
```

Comment:
isn't there allure constants for flow-ref? that would be mor appropriate here.

Reply:
what i mean is, the values of this annotations should match features of the runtime and stories for its users. spring is just an internally used library.

Reply:
Changed to Flow Reference Story